pkg/api/v1beta3/types.go

```diff
@@ -595,48 +593,48 @@ type PodStatusResult struct {
 // to hosts.
 type Pod struct {
 	TypeMeta   `json:",inline"`
-	ObjectMeta `json:"metadata,omitempty"`
+	ObjectMeta `json:"metadata,omitempty" description:"metadata specific to the Pod"`
 
 	// Spec defines the behavior of a pod.
-	Spec PodSpec `json:"spec,omitempty"`
+	Spec PodSpec `json:"spec,omitempty"description:"specification of the desired behavior of the pod"`
 
 	// Status represents the current information about a pod. This data may not be up
 	// to date.
-	Status PodStatus `json:"status,omitempty"`
+	Status PodStatus `json:"status,omitempty" description:"current status of the pod; populated by the system, read-only"`
```

Comment:
"mostly recently observed status of the pod..."

Also please add the link: https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/api-conventions.md#spec-and-status